src/core/connectivity.jl

```diff
@@ -63,6 +63,18 @@ struct LineConnectivity <: AbstractConnectivity
     end
 end
 
+function Base.isequal(c0::LineConnectivity, c1::LineConnectivity)::Bool
+    if c0.dimension != c1.dimension
+        return false
+    end
+
+    if c0.excluded_connections != c1.excluded_connections
+        return false
+    end
+
+    return c0.excluded_positions == c1.excluded_positions
```

Comment:
The `excluded_positions` are not sorted. They should be sorted in the `LineConnectivity` constructor.

Reply:
added